codec-http/src/main/java/io/netty/handler/codec/spdy/SpdyFrameCodec.java

```diff
@@ -32,7 +32,7 @@
  * A {@link ChannelHandler} that encodes and decodes SPDY Frames.
  */
 public class SpdyFrameCodec extends ByteToMessageDecoder
-        implements SpdyFrameDecoderExtendedDelegate, ChannelOutboundHandler {
+        implements SpdyFrameDecoderDelegate, ChannelOutboundHandler {
```

Comment:
Then you will need remove SpdyFrameDecoderExtendedDelegate.

Reply:
no this is still needed